packages/blade/src/components/Typography/Title/Title.tsx

```diff
@@ -0,0 +1,45 @@
+import type { ReactElement } from 'react';
+import type { TextTypes } from '../../../tokens/theme/theme';
+import getPlatformType from '../../../utils/getPlatformType';
+import BaseText from '../BaseText';
+import type { BaseTextProps } from '../BaseText/BaseText';
+
+export type TitleProps = {
+  variant: 'large' | 'medium' | 'small';
+  type: TextTypes;
+  children?: React.ReactNode;
+};
+
+const getProps = ({ variant, type }: TitleProps): BaseTextProps => {
+  const props: BaseTextProps = {
+    color: `surface.text.${type}.lowContrast`,
+    fontSize: 25,
+    fontWeight: 'bold',
+    fontStyle: 'normal',
+    lineHeight: 's',
+    fontFamily: 'text',
+  };
+
+  if (variant === 'small') {
+    props.fontSize = 600;
+    props.lineHeight = '4xl';
+    props.as = getPlatformType() !== 'react-native' ? 'h3' : undefined;
```

Comment:
Maybe we should instead check for `getPlatformType() === 'web'`

Only for web we will set h1,2,3 else undefined, otherwise for `unknown` platform too it will set h1,2,3

Reply:
it's browser or node(for ssr) so I'll have to write two conditions to evaluate this.

Reply:
Oh okay i see, fine then.

Reply:
made the changes, even I was not liking that previously :D